xbmc/cores/VideoPlayer/Interface/StreamInfo.h

```diff
@@ -73,6 +73,8 @@ struct VideoStreamInfo : StreamInfo
   std::string stereoMode;
   int angles = 0;
   StreamHdrType hdrType = StreamHdrType::HDR_TYPE_NONE;
+  int fpsRate{0};
+  int fpsScale{0};
```

Comment:
Can fps be negative? Maybe `unsigned int`is a better fit.

Reply:
done